File: features/lender/src/main/java/com/nivasafinance/features/lender/lenderoffice/dto/LenderOfficeRequestData.java
package com.nivasafinance.features.lender.lenderoffice.dto;

import com.nivasafinance.common.dto.AddressRequest;
import com.nivasafinance.features.lender.lenderoffice.enums.LenderOfficeStatus;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class LenderOfficeRequestData {
    private String name;
    private String key;
    private String lenderKey;
    private AddressRequest createAddressRequest;
    private LenderOfficeStatus status;
}

File: features/lender/src/main/java/com/nivasafinance/features/lender/lenderoffice/dto/LenderOfficeRequestData.kt
package com.nivasafinance.features.lender.lenderoffice.dto

import com.nivasafinance.common.dto.AddressRequest
import com.nivasafinance.features.lender.lenderoffice.enum.LenderOfficeStatus

data class LenderOfficeRequestData(
    val name: String,
    val key: String,
    val lenderKey: String,
    val createAddressRequest: AddressRequest?,
    val status: LenderOfficeStatus
)
